### ISSUE TITLE
TinyENC1 - I got it.

### ISSUE THREAD

**Issue by ppumkin** [matiaspl/LinkPi-ENCx#9]
Hello, Witam. 

I got sent your GitHub by somebody because I was having trouble with the 2603SE but I will also be getting the new TinyENC1 for demo purposes. 
You said contact you byt you have no contact details so I am opening this request. 
just email me on piotr at kula . solutions if you interested in looking the tiny.. and in the mean time I am going to get into the 2603SE with your guide.

Thanks

**Comment by matiaspl** [comment]
Dzień dobry,

Mogę rzucić w wolnej chwili okiem, jeśli temat nadal aktualny :)

**Comment by relaxibus** [comment]
Hello Piotr and Witam,
are you still working on the LinkPi stuff? I’m looking how could give me a online tutorial about the LinkSDK.

**Comment by matiaspl** [comment]
I can put some effort into getting root access to TinyENC, but I'd need the recovery firmware. The original factory image is hosted on some Chinese online platform that I was not able to register on.

**Comment by relaxibus** [comment]
If you send me the link I can ask a chinese colleague of my to register.

My goal is to create a modified firmware using the LinkSDK, this seems the most problematic part here. Any experience?

**Comment by ppumkin** [comment]
Hi all. 
Sorry but this got rid of it quite quickly as it wasn't really working for what we needed it for sadly 🙁

**Comment by relaxibus** [comment]
@ppumkin What is not really working?

**Comment by matiaspl** [comment]
@relaxibus [https://pan.baidu.com/s/18COoUgvCofjuo42rcbMeUg](https://gitee.com/link?target=https%3A%2F%2Fpan.baidu.com%2Fs%2F18COoUgvCofjuo42rcbMeUg) - it's mentioned here: https://gitee.com/LinkPi/Encoder/wikis/pages/preview?sort_id=2148530&doc_id=338777

**Comment by relaxibus** [comment]
@matiaspl @ppumkin Could you summarize what is not working for you?

**Comment by fu5ha** [comment]
Here's the zip :) Was able to (after some tribulation) get it downloaded. Seems to include their own version of a serial console and also has the base image of the tiny! https://drive.google.com/file/d/1NLsrJuJXJ_mI63C-IoBRNJ-It8sDJHJj/view?usp=drive_link

**Comment by matiaspl** [comment]
So there's definitely telnet there. One could try replacing the Encoder binary and libraries from ENC1 to ENC1 Tiny in hope of getting SRT... Not something for the faint-hearted, but on the other hand I think it's not easy to totally break it this way considered recovery procedure mentioned in the package instructions.
